.gitignore

```diff
@@ -1,18 +1,21 @@
 ###> symfony/framework-bundle ###
 /.env.local
+/.env.local.php
```

Comment:
dot php?

Reply:
Updated all files normally installed by flex. Decided to leave defaults in place.